tests/nemo_text_processing/zh/data_text_normalization/test_cases_measure.txt

```diff
@@ -2,5 +2,4 @@
 38°C~三十八摄氏度
 120m²~一百二十平方米
 10ms~十毫秒
-6.3%~百分之六点三

```

Comment:
same as above

Reply:
I moved it under the <fraction>. The updated grammar <fraction> grammar can process inputs with "%".